<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>Integrative Plasticity</title>
  <link href="http://netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.min.css" rel="stylesheet">
  <link rel="stylesheet" href="source/css/style.css">
  <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-43310906-1', 'plant-plasticity.github.io');
    ga('send', 'pageview');

  </script>
</head>
<body>

  <nav class="navbar navbar-fixed-top navbar-inverse">
    <div class="navbar-inner">
      <div class="container-fluid">
        <a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </a>
        <a href="index.html" class="brand">Integrative Plasticity</a>
        <div class="nav-collapse">
          <ul class="nav">
            <li class="active"><a href="#">About</a></li>
            <li><a href="aims.html">Aims</a></li>
            <li><a href="outreach.html">Outreach</a></li>
            <li><a href="software.html">Software</a></li>
            <li><a href="protocols.html">Protocols</a></li>
            <li><a href="people">People</a></li>
            <li><a href="publications.html">Publications</a></li>
          </ul>
        </div>
      </div>
    </div>
  </nav>

  <div class="container-fluid">
    <div class="hero-unit">
      <h1>Integrative analysis of plasticity in cell fate determination in plants.</h1>
      <hr>
      <a href="images/INTACT-2.png"><img class="pull-right img-rounded" src="images/INTACT-2.png" width=300 alt="p35S-INTACT in tomato hairy roots"></a>
      <p id="species"><strong>Species to be Studied:</strong> <em>Oryza sativa</em>, <em>Solanum lycopersicum</em>, <em>S. pennellii</em>, and <em>Medicago truncatula</em></p>
      <p><strong>Funded by:</strong> <a href="http://www.nsf.gov/funding/pgm_summ.jsp?pims_id=5338&amp;org=BIO" target="_blank">NSF Plant Genome Research Program</a> (DBI -1238243)</p>
      <!-- <h2>Abstract</h2> -->
      <section class="abstract">
        <p>From the germination of a seed to the fertilization events that lead to the next generation, plant development is exquisitely orchestrated by genetically determined processes that are fine-tuned by environmental cues. This entails the precise regulation of networks of genes in individual cells over the course of the plant life cycle.</p>
        <p>In this project, we will decipher the complex regulation of genes within specific types of the cells of the plant. This will be accomplished by transfer of methods our team has pioneered in the model plant species Arabidopsis thaliana to three important crops: rice, alfalfa and tomato. These technologies allow for the isolation of cell nuclei containing DNA and RNA and the ribosomes that translate mRNA into protein from targeted subpopulations of cells of a leaf, root or other organs, without the need for dissection.</p>
        <p>The new genetic resources developed will be used to study how development is perturbed by two major environmental threats to US agriculture: droughts and floods. The outcome will be a greater understanding of the integration of plant development with environmental cues. By use of parallel multi-tiered and computationally robust analyses, the project will address two important biological questions: How does gene regulation in the stem cells (meristem) of roots and shoots differ across species? How does environmental stress influence the development of specialized cell types in the root?</p>
        <p>The data generated will be disseminated through existing interactive websites and publications. The project will have multiple broader impacts. First, it will establish resources for the evaluation of cell-type specific expression in three important crops. The seed material and data sets will be shared with the plant genome research community through the <a href="http://www.ncbi.nlm.nih.gov/sra" target="_blank">NCBI Short Read Archive</a> and <a href="http://www.gramene.org/" target ="_blank">Gramene</a>. Second, the hypothesis driven experiments that address drought and flooding stress will provide broad new insights, which will facilitate downstream improvement of abiotic stress tolerance. Third, the project will engage postdoctoral researchers and graduate students in advanced interdisciplinary training in biology and computational sciences. These individuals will benefit from the self-confidence building experience of mentoring undergraduate students in research. Finally, the project will engage high school students in the classroom and the laboratory, develop teaching tools, and foster greater understanding of the importance of plant research to humankind.</p>
      </section>
      <h2>Practical Applications of Research</h2>
      <p>This project will establish technologies for multi-tier gene expression analyses in species of agronomic importance. The methods will be fine-tuned to interrogate activities of specific cell types that regulate plasticity of development in response to environmental stimuli, such as under extremes of water availability. Rich genetic and data resources will be provided to the community. Training of high school students through postdoctoral researchers in genomics, bioinformatics and cell biology will be enriched through mentoring and team research.</p>
      <h2>Experimental Approaches</h2>
      <p>We are advancing two technologies developed for Arabidopsis for microgenomic evaluation of developmental systems and environmental responses to three species of agronomic relevance. The method of nuclear isolation, INTACT (Isolation of Nuclei TAgged in specific Cell Types) enables genome-level profiling of nuclear poly(A)<sup>+</sup> mRNA (<em>nuclear transcriptome</em>), histone modifications (<em>epigenome</em>) and transcription factor binding.</p>
      <p>INTACT will be paired with the TRAP (Translating Ribosome Affinity Purification) system which enables profiling of poly(A)<sup>+</sup> mRNAs associated with ribosomes (<em>translatome</em>) and the occupancy of ribosomes on transcripts to estimate levels of protein synthesis. This involves use of specific promoters, such as those that define expression in specific cell types or regions of an organ, to drive expression of a nuclear membrane-targeted protein in the case of INTACT or a ribosomal protein in the case of TRAP.</p>
      <p>The resources developed will be tailored to the targeted species but may be more broadly applied. The stable transgenics will include genotypes that will enable INTACT or TRAP in multiple cell types (i.e., by use of quasi-constitutive promoters).</p>
    </div>

  </div>
  <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
  <script src="http://netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/js/bootstrap.min.js"></script>
</body>
</html>