---
layout: default
active: about
---

<body>

  {% include navbar.html %}

  <div class="container-fluid">
    <div class="hero-unit">
      <h1>Plasticity project</h1>
      <hr>
      <h2>RESEARCH-PGR - Adapting to a Harsh Environment: Arbuscular Mycorrhizal Fungi, Drought Stress and Plasticity of Plant Architecture for a Beneficial Outcome </h2>
      <p id="species"><strong>Species to be Studied:</strong> <em>Oryza sativa</em>, <em>Solanum lycopersicum</em>, and <em>Rhizophagus irregularis</em></p>
      <p><strong>Funded by:</strong> <a href="http://www.nsf.gov/funding/pgm_summ.jsp?pims_id=5338&amp;org=BIO" target="_blank">NSF Plant Genome Research Program</a> (IOS-1856749)</p>
      <p><strong>Co-PIs:</strong> <b>Julia Bailey-Serres</b>, University of California, Riverside; <b>Siobhan Brady</b>, University of California, Davis; <b>Neelima Sinha</b>, University of California, Davis; <b>Roger Deal</b>, Emory University; and <b>Uta Paszkowski</b>, Professor, Cambridge University, UK</p>
      <!-- <h2>Abstract</h2> -->
      <section class="abstract">
        <p>To meet the food and nutritional demands of an ever-growing population, food production must increase despite declines in arable land and limitations in key plant nutrients such as nitrogen and phosphorous (Pi), increased soil salinity, and predicted increases in the frequency of droughts. Meeting future food needs is one of the greatest challenges facing the world today, and can be addressed through crop improvement and soil management. Little is known of how plants integrate beneficial interactions with soil microbes and environmental limitations (i.e., insufficient water) to modulate their growth. This project is developing new technologies for studying the interaction between plants and microbes, focusing on how rice and tomato interact with beneficial fungi under varied irrigation conditions. The results will make it possible to improve plant-microbe interactions so that plants can better sustain vital functions under stressful conditions. The technologies will be broadly applicable to other aspects of plant-environmental interactions and to other multicellular organisms. In addition, the project is introducing students in local schools to topics in plant genetics that highlight the importance of beneficial organisms. </p>
        <p>This project seeks to identify key gene regulatory, signaling, and response networks that underlie beneficial plasticity in plant root development and shoot meristem activity mediated by the symbiosis with Arbuscular Mycorrhizal Fungi (AMF), particularly under water deficit conditions. The project assesses how AMF influences root architecture and meristematic development under low water availability. The specific aims are to (1) Develop inducible i-INTACT and i-TRAP to capture epigenome to translatome dynamics in the subset of cells responding to distinct AMF elicitors or developing arbuscules; (2) Initiate a high-resolution survey of AMF and water deficit interactions from the organ to the cellular level in the greenhouse; and (3) Pursue education and broader impact activities. This includes (i) prompt dissemination of genetic material, genomic data, protocols, software, and workshop materials; (ii) annual instruction of methods in courses, workshops and undergraduate teaching labs; and (iii) hands-on training of undergraduate and high school students in plant genomics research. Mentoring is tailored to each institutionâ€™s community, targeting underrepresented groups. The project also develops teaching materials to enhance the 3rd grade Next Generation Science Standard curricula through instruction on plant parts at the microscopic level, empowering youths with an appreciation of plants and the importance of plant research to humankind.</p>
      </section>     
      
      <hr>
      <a href="{{ site.imagesurl }}INTACT-2.png"><img class="pull-right img-rounded" src="{{ site.imagesurl }}INTACT-2.png" width=300 alt="p35S-INTACT in tomato hairy roots"></a>
      <h2>Integrative analysis of plasticity in cell fate determination in plants</h2>
      <p id="species"><strong>Species to be Studied:</strong> <em>Oryza sativa</em>, <em>Solanum lycopersicum</em>, <em>S. pennellii</em>, and <em>Medicago truncatula</em></p>
      <p><strong>Funded by:</strong> <a href="http://www.nsf.gov/funding/pgm_summ.jsp?pims_id=5338&amp;org=BIO" target="_blank">NSF Plant Genome Research Program</a> (IOS -1238243)</p>
      <p><strong>Co-PIs:</strong> <b>Julia Bailey-Serres</b>, University of California, Riverside; <b>Siobhan Brady</b>, University of California, Davis; <b>Neelima Sinha</b>, University of California, Davis; and <b>Roger Deal</b>, Emory University</p>
      <!-- <h2>Abstract</h2> -->
      <section class="abstract">
        <p>From the germination of a seed to the fertilization events that lead to the next generation, plant development is exquisitely orchestrated by genetically determined processes that are fine-tuned by environmental cues. This entails the precise regulation of networks of genes in individual cells over the course of the plant life cycle.</p>
        <p>In this project, we will decipher the complex regulation of genes within specific types of the cells of the plant. This will be accomplished by transfer of methods our team has pioneered in the model plant species Arabidopsis thaliana to three important crops: rice, alfalfa and tomato. These technologies allow for the isolation of cell nuclei containing DNA and RNA and the ribosomes that translate mRNA into protein from targeted subpopulations of cells of a leaf, root or other organs, without the need for dissection.</p>
        <p>The new genetic resources developed will be used to study how development is perturbed by two major environmental threats to US agriculture: droughts and floods. The outcome will be a greater understanding of the integration of plant development with environmental cues. By use of parallel multi-tiered and computationally robust analyses, the project will address two important biological questions: How does gene regulation in the stem cells (meristem) of roots and shoots differ across species? How does environmental stress influence the development of specialized cell types in the root?</p>
        <p>The data generated will be disseminated through existing interactive websites and publications. The project will have multiple broader impacts. First, it will establish resources for the evaluation of cell-type specific expression in three important crops. The seed material and data sets will be shared with the plant genome research community through the <a href="http://www.ncbi.nlm.nih.gov/sra" target="_blank">NCBI Short Read Archive</a> and <a href="http://www.gramene.org/" target ="_blank">Gramene</a>. Second, the hypothesis driven experiments that address drought and flooding stress will provide broad new insights, which will facilitate downstream improvement of abiotic stress tolerance. Third, the project will engage postdoctoral researchers and graduate students in advanced interdisciplinary training in biology and computational sciences. These individuals will benefit from the self-confidence building experience of mentoring undergraduate students in research. Finally, the project will engage high school students in the classroom and the laboratory, develop teaching tools, and foster greater understanding of the importance of plant research to humankind.</p>
      </section>
      <h3>Practical Applications of Research</h2>
      <p>This project will establish technologies for multi-tier gene expression analyses in species of agronomic importance. The methods will be fine-tuned to interrogate activities of specific cell types that regulate plasticity of development in response to environmental stimuli, such as under extremes of water availability. Rich genetic and data resources will be provided to the community. Training of high school students through postdoctoral researchers in genomics, bioinformatics and cell biology will be enriched through mentoring and team research.</p>
      <h3>Experimental Approaches</h2>
      <p>We are advancing two technologies developed for Arabidopsis for microgenomic evaluation of developmental systems and environmental responses to three species of agronomic relevance. The method of nuclear isolation, INTACT (Isolation of Nuclei TAgged in specific Cell Types) enables genome-level profiling of nuclear poly(A)<sup>+</sup> mRNA (<em>nuclear transcriptome</em>), histone modifications (<em>epigenome</em>) and transcription factor binding.</p>
      <p>INTACT will be paired with the TRAP (Translating Ribosome Affinity Purification) system which enables profiling of poly(A)<sup>+</sup> mRNAs associated with ribosomes (<em>translatome</em>) and the occupancy of ribosomes on transcripts to estimate levels of protein synthesis. This involves use of specific promoters, such as those that define expression in specific cell types or regions of an organ, to drive expression of a nuclear membrane-targeted protein in the case of INTACT or a ribosomal protein in the case of TRAP.</p>
      <p>The resources developed will be tailored to the targeted species but may be more broadly applied. The stable transgenics will include genotypes that will enable INTACT or TRAP in multiple cell types (i.e., by use of quasi-constitutive promoters).</p>
    </div>

  </div>
</body>
